src/const.ts

```diff
@@ -1 +1,41 @@
-export const RUNE_TICKER = 'RUNE'
+import { getAssetFromString, assetToBase, assetAmount } from '@thorchain/asgardex-util'
+
+import {
+  PricePoolCurrencySymbols,
+  PricePoolCurrencyWeights,
+  PricePoolAssets,
+  PoolAsset,
+  PricePool
+} from './views/pools/types'
+
+// Rune ticker as const - just because we use it almost everywhere ...
+export const RUNE_TICKER = getAssetFromString(PoolAsset.RUNE)?.ticker ?? 'RUNE'
+
+// Currency symbols used for pricing
+export const CURRENCY_SYMBOLS: PricePoolCurrencySymbols = {
+  [PoolAsset.RUNE]: 'ᚱ',
+  [PoolAsset.BTC]: '₿',
+  [PoolAsset.ETH]: 'Ξ',
+  [PoolAsset.TUSDB]: '$'
+}
+
+// Weight of currencies to use for pricing in application
```

Comment:
Typo: `Weight of currencies to use for pricing in application` -> `Weight of currencies needed for pricing`